src/ctapipe/calib/camera/calibrator.py

```diff
@@ -283,7 +298,11 @@ def _calibrate_dl1(self, event, tel_id):
             )
 
             # correct non-integer remainder of the shift if given
-            if self.apply_peak_time_shift.tel[tel_id] and remaining_shift is not None:
+            if (
+                self.apply_peak_time_shift.tel[tel_id]
+                and remaining_shift is not None
+                and self.image_calibration_type == "charge"
```

Comment:
Do we really need to alter this condition?

Reply:
Just check if `dl1.peak_time is not None` instead of a specific extractor.

Reply:
@maxnoe that makes sense